### ISSUE TITLE
[ERROR]: Failed to decrypt OMEMO message - OmemoHelper is null 

### ISSUE THREAD

**Issue by o---** [UWPX/UWPX-Client#134]
## Bevor you create a new issue:

```
- [X] I searched for similar issues and did not find one
- [X] I'm using the latest version available in the [Windows Store](https://www.microsoft.com/store/apps/9NW16X9JB5WV)
```

## I'm submitting a...:

- Bug report (I searched for similar issues and did not find one) 
 

- Question or Support request

## Current behavior:

**I am not sure how to get OMEMO working.** I have exchanged a bunch of messages (unencrypted), but as soon as switch on encryption, I am not able to send nor receive messages nor message carbons. 

I am finding following messages in the log:
```
[ERROR][27.05.2020 22:15:28]: Failed to decrypt OMEMO message - OmemoHelper is null 
[ERROR][27.05.2020 22:15:51]: Failed to send OMEMO message - OmemoHelper is null 
[ERROR][27.05.2020 22:16:58]: Failed to send OMEMO message - OmemoHelper is null 
[ERROR][27.05.2020 22:17:36]: Failed to decrypt OMEMO message - OmemoHelper is null 
[ERROR][27.05.2020 22:17:41]: Failed to send OMEMO message - OmemoHelper is null 
[ERROR][27.05.2020 22:18:50]: Failed to load OMEMO prekeys for: my.account@domain.com 
```

I have checked if I see any un-trusted OMEMO keys, but all are set to trusted. 
![image](https://user-images.githubusercontent.com/5457048/83069004-8022dc80-a069-11ea-91d7-7b3dcf179def.png)

![image](https://user-images.githubusercontent.com/5457048/83068981-75684780-a069-11ea-9138-9537bd9e2284.png)

## Expected behavior:
OMEMO messages are working. No errors in the log. 

## Minimal reproduction of the problem with instructions:

1. Enable OMEMO
2. Exchange some messages

## Environment:

```
App Version(s): 
v.0.26.2.0 <!-- Which version of the App are you using (e.g. v.0.2.0) -->

Windows 10 Version Number: <!-- https://en.wikipedia.org/wiki/Windows_10_version_history -->
- [X] 1909
- [ ] 1809
- [ ] 1803
- [ ] 1709
- [ ] 1703
- [ ] 1607
- [ ] 1511
- [ ] 1507
- [ ] Insider Build (build number: )
- [ ] Misc:

Device form factor:
- [X] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub

Where did you got the APP from?
- [X] [Windows Store](https://www.microsoft.com/store/apps/9NW16X9JB5WV)
- [ ] Self-build, using a provided release source
- [ ] Self-build, repo cloned at [dd.mm.yyy] <!-- When did you clone the repo! -->
- [ ] Misc, got it from... <!-- Please tell us your source! -->

```

**Comment by COM8** [comment]
This looks to me like the way OMEMO keys are being stored is still very much broken...
I need to refactor this.

**Comment by o---** [comment]
I guess these are also related to this topic:
```
[ERROR][29.05.2020 00:28:21]: OmemoStore: No such signed pre key: 5 
[ERROR][29.05.2020 00:28:21]: Failed to load OMEMO signed prekey pair for: another.account@domain.com 
[ERROR][29.05.2020 00:28:21]: OmemoStore: No such signed pre key: 5 
[ERROR][29.05.2020 00:28:21]: Failed to load OMEMO signed prekey pair for: my.account@domain.com 
```

**Comment by COM8** [comment]
Yes. This is definitely caused by UWPX loosing keys...

**Comment by COM8** [comment]
Fixed by #146.
